src/Integration/Binding/CSharpVBBindingOperation.cs

```diff
@@ -127,19 +127,12 @@ private void AddRuleset()
 
         private void AddAdditionalFile()
         {
-            try
-            {
-                var projectSystem = serviceProvider.GetService<IProjectSystemHelper>();
-                projectSystem.AssertLocalServiceIsNotNull();
-                // It's possible that the additional file is set property, but the ruleset is out of date, which will trigger the binding. So we need to remove the file first, otherwise we get an error that the file already exists.
-                // It's possible that the additional file exists but has a wrong type, e.g. Compile/Content, and not AdditionalFiles. In which case removing and re-adding it solves the problem.
-                projectSystem.RemoveFileFromProject(initializedProject, cSharpVBBindingConfig.AdditionalFile.Path);
-                projectSystem.AddFileToProject(initializedProject, cSharpVBBindingConfig.AdditionalFile.Path, "AdditionalFiles");
-            }
-            catch (Exception ex)
-            {
-                throw new Exception($"Failed to bind project {initializedProject.FullName}, exception details: {ex}");
-            }
+            var projectSystem = serviceProvider.GetService<IProjectSystemHelper>();
```

Comment:
try-catch removal

Reply:
Any file system operation either direct or indirect via the DTE could potentially throw.
Having said that, we should have a try-catch block higher up to catch exceptions during binding. I'll check.